zebra/rtadv.c

```diff
@@ -467,8 +467,8 @@ static int rtadv_timer(struct thread *thread)
 	struct zebra_if *zif;
 	int period;
 
-	zrouter.rtadv.ra_timer = NULL;
-	if (zrouter.rtadv.adv_msec_if_count == 0) {
+	zns->rtadv.ra_timer = NULL;
```

Comment:
If RA is not owned by the zrouter than it is owned by the zvrf *not* the zebra namespace object.  As outlined in the past the zns data structure is a support structure for vrf's.

Reply:
I will do it.

Reply:
done. please review.